web/src/main/resources/META-INF/resources/webjars/common/webcert/utils/http403ResponseInterceptor.service.js

```diff
@@ -68,18 +68,23 @@ angular.module('common').provider('common.http403ResponseInterceptor',
                         // if we aren't allowed to navigate we are most likely djupintegrerade.
                         //TODO: use sessionType or something better than NAVIGERING to determine
                         //which redirectUrl to use.
-                        if (!authorityService.isAuthorityActive({authority: 'NAVIGERING'})) {
+                        if (response.data && response.data.message && response.data.message === 'Invalid launchId') {
+                            redirectUrl += 'another-session-active';
```

Comment:
Consider renaming the error reason 'another-session-active'. Reason for the error is because the launchId isn't valid and the webclient is therefor invalid.

Maybe just as simple as  'invalid-launchid'.